Jenkinsfile

```diff
@@ -27,13 +27,12 @@ node {
     println "Building static agent docker image"
     buildImage(STATIC_IMAGE_NAME, "static", releaseVersion, "${DYNAMIC_IMAGE_NAME}:${releaseVersion}")
   }
-  
+
   stage ('Build RHEL Agent Docker Image') {
-    //TODO fix redhat build
-    //if (env.RHEL.toBoolean()) {
-    //  println "Building rhel agent docker image"
-    //  buildImage(RHEL_IMAGE_NAME, "rhel", releaseVersion, null)
-    //}
+    if (env.RHEL.toBoolean()) {
```

Comment:
do we still need this safe guard? I suggest also building the REHL image every time we build the "normal" images.

Reply:
would also handle it the same way

Reply:
Alrighty, I'll remove the safeguard, just a moment